### ISSUE TITLE
Incorrect version of SLSO for SexLabFramework installed

### ISSUE THREAD

**Issue by tb88a1** [ForgottenGlory/Dungeons-Deviousness#237]
**D&DDC Version**
2.3.5

**Describe the bug**
SLSO does not automatically advance to the next stages, and orgasm never occurs, even when the arousal level is at 100%

**To Reproduce**
Start a new sex scene, preferably with the arousal level at 100%

**Expected behavior**
the stages should automatically advance, and possibly orgasm when reaching 100% in the widget

Looking at the documentation of SLSO (1.7.4), it says that the installed version is only compatible with SexLabFramework SE 163 (not beta) and version 164b, however, the installed version of SexLabFramework SE is 163b9 on D&DDC 2.3.5 (not compatible).

If you check the sslThreadController.psc file for this version of SLSO and the previous version, you will see that calls to the SexLab functions have changed.
The same is true for the sslActorAlias.psc and sslBaseVoice.psc files (I didn't check all the files).

To make sure of this, I manually updated SexLabFramework SE version 163b9 to version 163 (final), and SLSO worked fine.

I don't know if this update will break other mods.

**Comment by tb88a1** [comment]
according to the documentation of all mods in the "SexLab & Friends" group, almost none (with the exception of SLSO, and perhaps SLAX*) require a newer version of the SexLab Framework (the vast majority of MODs do not have recent updates) .

*I don't know which MOD and version is SLAX (SexLab Aroused? - If SexLab Aroused Creature, link to SL pointed to version 169b2).

**later (if I have time) I'll check if SexLabFramework removed some functions in this new version, or SLSO started using functions that didn't exist before.

**Comment by tb88a1** [comment]
I made some tests with different versions of SexLabFramework SE and SexLab Separate Orgasm.

**How I made the tests:**
- Enabled the Papyrus logs (but nothing relevant was found);
- Started a new game, with all all MCM options according to the documentation (MCM documentation);
- Using G Menu, selected "Self" -> "Play with Myself" -> "Tease" (until reach 100%);
- Used the magic "SexLab Debug (Target)" and selected the same NPC on every tests;

---

### #Test 01:
It's the D&DDC version 2.3.5 without any modification
This tests use the following mods:
- SexLab Framework SE v163b9;
- SexLab Separate Orgasm version 1.7.4.1.

**Results: BUG**
**Automatically advances to the next stage: NO**
**Can orgasm: NO**

---

### #Test 02:
It's basically the previous version of D&DDC.
This tests use the following mods:
- SexLab Framework SE v163b9;
- SexLab Separate Orgasm version 1.7.4 (downgraded - not available on LoversLab);


**Results: WORKS**
**Automatically advances to the next stage: YES**
**Can orgasm: YES**

---

### #Test 03:
I just upgrade manually the SexLab Framework SE to the latest non-beta version.
This tests use the following mods:
- SexLab Framework SE v163 (upgrade to the latest non-beta version);
- SexLab Separate Orgasm version 1.7.4.1;


**Results: WORKS**
**Automatically advances to the next stage: YES**
**Can orgasm: YES**

---

### #Test 04:
I upgrade manually the SexLab Framework SE to the latest non-beta version and downgraded the SLSO to previous version.
This tests use the following mods:
- SexLab Framework SE v163 (upgrade to the latest non-beta version);
- SexLab Separate Orgasm version 1.7.4 (downgraded - not available on LoversLab);


**Results: WORKS**
**Automatically advances to the next stage: YES**
**Can orgasm: YES**

---

**Conclusions:**
The actual version of D&DDC has a bug on modlist, and the problem is not SLSO, but the versions that are installed (SexLab Framework and SLSO).
The SLSO documentation has in requirements the following message: _"exactly Sex Animation Framework v1.6XXX"_

To fix, **it's necessary to update SexLab Framework SE** to the latest non-beta version (or beta, but SLSO must use the same version - available on LoversLab);

In **#Test 04**, the SexLab Framework was updated, and SLSO was downgraded and even so it kept working, so can it be assumed that the Sexlab Framework maintains backwards compatibility and doesn't break other mods? (not 100% sure)


**Possible Solution:**
**Update the SexLab Framework SE**

**Comment by dervishxgit** [comment]
This is an exemplary test, thorough, and exactly what one would ask for. Thank you.

**Comment by dervishxgit** [comment]
I have replicated these steps and results and can confirm satisfaction.

**Comment by dervishxgit** [comment]
I've tested the SL 163 (not beta) for a few hours in a starting playthrough and have found the following.

Nutshell: basic scenes, orgasm, and consequences (like cum and other effects) seem to work. The setup process did change. Screenshots will be uploaded to discord showing differences.

Tested successfully in game:
- player orgasm in solo and pair scenes 
- partner orgasm 
- normal sl behavior and all mods calling sl (cursed loot, battle fuck, radiant prostitution)
- secondary effects and consequences like cum, orgasm fatigue

Differences observed in setup process:
- Sexlab does not automatically start installing itself upon reload. MCM interaction was required to kickstart the process. SL setup steps were otherwise normal and consistent with readme.
- Sexlab tools mcm appeared not to accept input (like keybind changes), until after it was opened, closed, and reopened again once at start up, similar to a few other tempermental mcms. 
- SL debug spells not present, presumably because this is a release, not debug version
- Almost no console output visible, likely for the same reason

Conclusion: Aside from differences in setup potentially requiring readme updates, this SL and related mods perform as expected. We may still have interest in the 163b version if the debug behaviors are an asset.
